codecov.yml

```diff
@@ -0,0 +1,31 @@
+codecov:
+  notify:
+    after_n_builds: 10 # Keep in sync with .github/workflows/ci.yml
+
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 5%
```

Comment:
Yes it is, it allows for up to 5% of coverage decrease before turning the line red :) Considering our current codebase size, if test coverage decreases by more than 5% in a PR then something has gone really wrong, so I thought I’d leave this check just in case.

Reply:
Also, we’re still setting `patch` requirements that will turn the check red if the patch is less well-covered than the average of our codebase. This might be cumbersome when doing eg. refactorings, but OTOH if we make sure to always keep these checks green it’ll help us slowly add tests.

I think the most interesting thing is the split between unit tests and other tests, so that we have a different number for counting only the code coverage by unit tests, which is the one we actually want to ramp up the most.